### DISCUSSION TITLE
until unless the model loads show something like a spinner

### DISCUSSION THREAD

**Discussion by olumolu** [Jeffser/Alpaca#343] [General]
Showing some spinner would have been a good option as it will give the user the knowledge that the model is loading cor larger models i find it take many seconds and you won't be able to understand what is going on. As the model is still loading.

**Comment by Jeffser** [comment]
![image](https://github.com/user-attachments/assets/3c8bfc4b-6035-436d-992f-bf2d8be850f1)

What about the spinner that exists already when the user sends a message?

**Reply by olumolu** [reply]
Yes it is best while loading the model without sending massage lets show a spinner

**Reply by Jeffser** [reply]
> Yes it is best while loading the model without sending massage lets show a spinner 

The model can only be loaded once the user has sent a message, that's how Ollama works

**Comment by olumolu** [comment]
no more relevant
